airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -544,27 +547,34 @@ public void testIncrementalSync() throws Exception {
     LOGGER.info("Starting testIncrementalSync() sync 2");
     final JobInfoRead connectionSyncRead2 = apiClient.getConnectionApi()
         .syncConnection(new ConnectionIdRequestBody().connectionId(connectionId));
-    waitForSuccessfulJob(apiClient.getJobsApi(), connectionSyncRead2.getJob());
+    // waitForSuccessfulJob(apiClient.getJobsApi(), connectionSyncRead2.getJob());
+    final JobRead job =
+        waitForJob(apiClient.getJobsApi(), connectionSyncRead2.getJob(),
+            Sets.newHashSet(JobStatus.PENDING, JobStatus.RUNNING, JobStatus.INCOMPLETE, JobStatus.FAILED));
```

Comment:
why is this changing? don't we want to wait for it to complete instead of to just be in running.

Reply:
We have changed the behavior of the reset, it is now cancelling a running job instead of waiting for it to terminate. That's why we are also expecting to have failed status.

Reply:
I add another look and this was a mistake, I restored the previous way to do it.

Reply:
:+1: